sled-agent/src/bootstrap/server.rs

```diff
@@ -93,17 +83,19 @@ impl Server {
         .map_err(|error| format!("initializing server: {}", error))?
         .start();
 
-        let inner_log =
-            log.new(o!("component" => "rack init server (BootstrapAgent)"));
-        let rack_init_server_handle = Inner::start_rack_init_server(
-            Arc::clone(&bootstrap_agent),
-            inner_log,
-        )
-        .await?;
+        // Start the currently-misnamed sprockets server, which listens for raw
+        // TCP connections (which should ultimately be secured via sprockets).
+        let sprockets_server =
+            SprocketsServer::bind(Arc::clone(&bootstrap_agent), &log)
```

Comment:
part of me wonders if the complexity of managing this raw TCP server means we'd be better served by "just using a dropshot interface" here, but that's a thing we can think about / clean-up elsewhere.